src/nu/xom/samples/SimpleTransformer.java

```diff
@@ -0,0 +1,46 @@
+/* Simple XSLT transformer using JDK built-in processor */
+package nu.xom.samples;
+
+import nu.xom.Builder;
+import nu.xom.Document;
+import nu.xom.Node;
+import nu.xom.Nodes;
+import nu.xom.xslt.XSLTransform;
+
+public class SimpleTransformer {
```

Comment:
No. There should be no additions to the core XOM API at all. This is build tooling change, not new published API.

If this is needed it needs to go somewhere else.